src/middlewared/middlewared/etc_files/local/apache24/Includes/webdav.conf

```diff
@@ -16,12 +14,16 @@
 	if not os.path.isdir(oscmd):
 		os.mkdir(oscmd, 0o774)
 
-	# Changing permissions for all directories and files in oscmd
-	uid = pwd.getpwnam('webdav').pw_uid
-	gid = grp.getgrnam('webdav').gr_gid
+	# Changing ownership for all directories and files in oscmd
+	webdav_user = middleware.call_sync('user.query', [['username', '=', 'webdav']])
+	if not webdav_user:
+		uid, gid = 0, 0
+	else:
+		uid, gid = webdav_user[0]['uid'], webdav_user[0]['group']['bsdgrp_gid']
+
 	for root, dirs, files in os.walk(oscmd):
 		for o in list(itertools.chain(dirs, files)):
-			os.chmod(os.path.join(root, o), uid, gid)
+			os.chown(os.path.join(root, o), uid, gid)
```

Comment:
You know, all this chown part in config generation doesnt like make sense to me.

How about we remove it all from here to middlewared on webdav.update?